fvcore/common/checkpoint.py

```diff
@@ -258,18 +258,26 @@ class PeriodicCheckpointer:
     multiple of period or if `max_iter` is reached.
     """
 
-    def __init__(self, checkpointer: Any, period: int, max_iter: int = None):
+    def __init__(
+        self,
+        checkpointer: Any,
+        period: int,
+        max_iter: int = None,
+        max_to_keep: int = None,
+    ):
         """
         Args:
             checkpointer (Any): the checkpointer object used to save
             checkpoints.
             period (int): the period to save checkpoint.
             max_iter (int): maximum number of iterations. When it is reached,
                 a checkpoint named "model_final" will be saved.
+            max_to_keep (int): maximum number of checkpoints to keep
```

Comment:
Could you make it more explicit that the oldest checkpoints will be deleted? Thanks!